src/app/qgsrecentprojectsmenueventfilter.cpp

```diff
@@ -0,0 +1,77 @@
+/***************************************************************************
+               qgsrecentprojectsmenueventfilter.cpp
+               ----------------------------------------------------
+    begin                : August 2023
+    copyright            : (C) 2023 by Yoann Quenach de Quivillic
+    email                : yoann dot quenach at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsrecentprojectsmenueventfilter.h"
+
+#include "qgswelcomepage.h"
+#include "qgsprojectlistitemdelegate.h"
+
+#include <QMenu>
+#include <QAction>
+#include <QEvent>
+#include <QMouseEvent>
+
+
+QgsRecentProjectsMenuEventFilter::QgsRecentProjectsMenuEventFilter( QgsWelcomePage *welcomePage, QObject *parent )
+  : QObject( parent ), mWelcomePage( welcomePage )
+{
+}
+
+bool QgsRecentProjectsMenuEventFilter::eventFilter( QObject *obj, QEvent *event )
+{
+
+  if ( event->type() != QEvent::MouseButtonPress )
+    return false;
```

Comment:
Instead of returning false, we should pass on to the base class:

```suggestion
    return QObject::eventFilter( obj, event );
```

See https://github.com/KDE/clazy/blob/master/docs/checks/README-base-class-event.md#base-class-event

Reply:
Ok